Updated docs to show responsibility of data owners (#1273)
ebi-ait/hca-ebi-wrangler-central: docs/SOPs/Triage/GDPR_Guidelines.md

## Summary diagram

```mermaid
graph TD
    A[What is the source of the project?]
    A -->|Published Paper| B{Does the project\nhave living donors?}
    A -->|Contributor\nCommunications| C{Is the project\narchived?}
    C -->|Yes| B
    C -->|No| G{Does the project\nhave living donors?}
 
    B -->|Yes| E{"Is the data and metadata\nopen access in the public domain?\n(ENA, AE, publication)"}
    B -->|No| F[We can curate it\nwith no worries]
    
    E -->|Yes| H[We can curate\nwhatever is in the public domain]
    E -->|No| R{Ask contributor\nabout consent they hold}
    
    R -->|Must be open acccess\nwith only these data?| I[Wrangle\nonly the open access\ndata & metadata:\nproject-level data\nanalysis protocols\nand analysis files]

>     R -->|consent\nfor more open data?| J[Contributor\ncan provide written consent\nthat they are aware that data\nwill be publicly open]

Reviewer: ```suggestion
    R -->|consent\nfor more open data?| J[Contributor\ncan provide written consent\nto release metadata\nas open access]
```

    R -->|Must be managed access?| K[Follow the\nManaged Access route\nto archive\nwith controlled access]

    H --> L[Be aware of obtaining\nextra metadata or data\non existing living donors\nnot publicly available\nas that may be subject\nto data protection]

>     H -->|consent\nfor more open data?| J[Contributor\ncan provide written consent\nthat they are aware that data\nwill be publicly open]

Reviewer: ```suggestion
    H -->|consent\nfor more open data?| J[Contributor\ncan provide written consent\nto release metadata\nas open access]
```

When wrangling **contributor led** publications, wranglers cannot recommend or suggest the contributors to archive their data in an open database if living donors are involved. Wranglers are also not allowed to state to the contributors that they could take the data and metadata from an open database once it's published there, as it might lead the contributors to make their data publicly available. **HCA recommendation when dealing with living donors should always be managed access**

### Metadata

> When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are allowed to take the information that the authors have made already of **public domain** or the authors provide more metadata that what's already publicly available but also provide **written consent** for its public release then the additional metadata can be wrangled as open access. ~~Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. ~~

Reviewer: ```suggestion
When wrangling from already **published projects**, wranglers should **never ask for extra donor metadata** if it involves living donors. Wranglers are allowed to take the information that the authors have made already of **public domain**. Wranlger may include previously unreleased metadata provided by the author only if the author gave  **written consent** for its public release as open access. ~~Moreover, to prevent low *k-anonymity* [^1] levels, wranglers should **avoid wrangling donor metadata when the project includes less than 5 living donors**. ~~
```
